zcash_primitives/src/transaction/fees/zip317.rs

```diff
@@ -78,13 +78,7 @@ impl FeeRule {
     ///
```

Comment:
```suggestion
    ///
    /// Using this fee rule with
    /// ```compile_fail
    /// marginal_fee < 5000 || grace_actions < 2 \
    ///     || p2pkh_standard_input_size > P2PKH_STANDARD_INPUT_SIZE \
    ///     || p2pkh_standard_output_size > P2PKH_STANDARD_OUTPUT_SIZE
    /// ```
    /// violates ZIP 317, and might cause transactions built with it to fail.
    ///